test/spec/tasks/check_enabled_tests.ex

```diff
@@ -0,0 +1,58 @@
+defmodule Mix.Tasks.CheckEnabledTests do
+  @moduledoc """
+  A task that checks which spec-tests are enabled or skipped. Useful to make sure that desired
+  tests are running.
+
+  It traverses vector configs, forks and runners hierarchically.
+
+  Run with MIX_ENV=test mix check_enabled_tests
+  The output will be in a reports/spec_test_report.exs file, that can be formatted using mix format.
+  """
+
+  require Logger
+
+  alias Spec.MetaUtils
+
+  def run(_args) do
+    MetaUtils.check_enabled() |> print(0)
+    :ok
+  end
+
+  defp print({name, report}, spaces) do
+    print_spaced(name, spaces)
+    new_spaces = spaces + String.length(name)
```

Comment:
To account for long names, I think it's better to use a fixed size here.

```elixir
    new_spaces = spaces + 4  # may be a module attribute
```

Reply:
Added 3-space indents + 🟡 emoji for the partial runs. This way, the partial runs are aligned with other titles and lines are not that long.